master/setup.py

```diff
@@ -453,7 +453,7 @@ def define_plugin_entries(groups):
     'txaio ' + txaio_ver,
     'autobahn ' + autobahn_ver,
     'PyJWT',
-    'distro'
+    'distro;platform_system==="Linux"'
```

Comment:
This was defined in https://www.python.org/dev/peps/pep-0508/. Based on this post https://discourse.numenta.org/t/setup-py-error-invalid-environment-marker/1298/3 it's supported by:

 - setuptools: 20.2.2 +
 - wheel: 0.24.0 +
 - pip: 8.1.2 +